src/models/api/DeploymentResponse.ts

```diff
@@ -18,6 +19,8 @@ export type DeploymentResponse = {
   flow_id: string,
   schedule: ScheduleResponse | null,
   is_schedule_active: boolean,
+  paused: boolean,
+  schedules: DeploymentScheduleResponse[],
```

Comment:
There's an OSS PR to add them, just needs to get merged, but that's a good reminder.

Reply:
Cool! Just a reminder/word-of-caution on timing here then - _if_ this gets merged before these new fields are available/defined everywhere, then it's quite _possible_ that we run into runtime errors that TS isn't seeing atm. 

For example anything like `deployment.schedules.map(...)` which thinks schedules is _always_ a defined array will go boom at runtime and TS won't see it at all. 


alternatively you could handle that this could be undefined and then in the mapper you can coalesce it. 
otherwise just pay close attention to order of operations here.

Reply:
The OSS side of this is merged now.